analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Sonar/InvalidCastToInterface.CSharp10.cs

```diff
@@ -1,5 +1,7 @@
 ﻿public interface IMyInterface { }
 
+public class SomeImpl : IMyInterface { }
```

Comment:
Ok, definitely there is nothing to be done from the point of our current, ongoing sprint. But shouldn`t the rule be updated? I do not really understand WHY is a concrete implementation needed in order to have an issue reported?

Reply:
I can tell why that is, but the behavior is explicitly called out in the RSpec https://sonarsource.github.io/rspec/#/rspec/S1944/csharp

> Exceptions
> No issue is reported if the interface has no implementing class in the assembly.

I agree with you that this limitation is nonsensical. Should we create an issue?

Reply:
I suggest to start a discussion on this topic on our slack channel, maybe others see a reason why this exception exists. If not, we can open an issue to fix it in the future.